openapi/index.yaml

```diff
@@ -399,23 +578,83 @@ definitions:
     pattern: "[0-9]{4}-[0-9]{2}-[0-9]{2}"
     x-example: "2019-09-15"
   ProblemJson:
-    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v7.3.0/openapi/definitions.yaml#/ProblemJson"
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v14.5.0/openapi/definitions.yaml#/ProblemJson"
```

Comment:
can we upgrade to 14.6.0?

Reply:
Fixed in [469e173](https://github.com/pagopa/io-functions-services/pull/67/commits/469e173201722102ed04c60b05300f6618a02fe2)